src/components/Events/SingleEvent.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+
+  <div class="event-container">
+    <router-view></router-view>
+      <div class="event-name">
+        <h1>{{ singleEvent.name }}</h1>
+      </div>
+      <div class="event-description">
+        <p>Event Description: {{ singleEvent.description }}</p>
+      </div>
+      <div class="event-buttons">
+        <button>Register</button>
+        <router-link to="events">
+            <button>Back to Events</button>
+        </router-link>
+      </div>
+      <div class="event-info">
+        <p>Location: {{ singleEvent.location }}</p>
+        <p>Time: {{ singleEvent.time }}</p>
+        <p>What to Bring: {{ singleEvent.bring }}</p>
```

Comment:
can you name this field something more descriptive than bring like "materialsRequired"